### PR TITLE
Added Products page

### PR DESCRIPTION
**Pull request by zoyatanweer**

### CODE REVIEW

File: components/products/products.css [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    padding: 2rem 1rem 5px 1rem;
+}
+.left-filter-title{
+    font-size: 1.8rem;
+    font-weight: 500;
+}
+.right-clear-btn{
+    color: var(--primary-color);
+    font-size: 1.7rem;
+    background-color: #f8f9fa;
```
Comment (DivyanshSareen) [inline components/products/products.css]: var for color

File: css/main.css [hunk 2]
```diff
@@ -44,14 +44,15 @@ picture {
 :root{
     --primary-color: #f72585;
     --secondary-color: #ff4d6d;
-    --background-color: #f0ecec;
-    --navigation-container-bg: #f8f9fa;
+    --background-color: #f8f9fa;
+    /* --navigation-container-bg: #f8f9fa; */
```
Comment (DivyanshSareen) [inline css/main.css]: remove comment if not for future use

**Review by DivyanshSareen — APPROVED** [review]
Looks good. Good to merge 👍

**Review by dev-prabhat — APPROVED** [review]
UI is clean , good to go

**Review by zoef-yt — APPROVED** [review]
Nice images. Looks good to me. Good to merge.

**Review by Muzammil-khan710 — APPROVED** [review]
Looking great !. Good to merge !

**Comment by zoyatanweer** [comment]
Thankyou guys!

**Pull request merged**